src/store/modules/stores.spec.js

```diff
@@ -39,10 +32,15 @@ describe('stores module', () => {
     store.commit('stores/Receive Stores', { stores: [store1, store2, store3] })
   })
 
-  it('sets routeError if store is not accessible', () => {
+  it('throws routeError if store is not accessible', async () => {
+    expect.assertions(2)
     mockGet.mockImplementationOnce(throws(createValidationError({ detail: 'Not found' })))
-    store.dispatch('stores/selectStore', 9999)
-    expect(routeError.actions.set).toBeCalled()
+    try {
+      await store.dispatch('stores/selectStore', { storeId: 9999 })
+    }
+    catch (e) {
+      expect(e.type).toEqual('RouteError')
+    }
```

Comment:
This does not error on the case where the error is not thrown, you can do it with this:

```js
await expect(store.dispatch('stores/selectStore', { storeId: 9999 })).rejects.toHaveProperty('type, 'RouteError')
```